packages/payment-detection/test/erc777/superfluid-retriever.test.ts

```diff
@@ -50,10 +50,51 @@ describe('api/erc777/superfluid-info-retriever', () => {
       expect(transferEvents[0].parameters?.txHash).toEqual(paymentData.txHash);
       expect(transferEvents[0].parameters?.block).toEqual(paymentData.block);
     });
+
+    it('should get payment events from SuperFluid via subgraph with 1 request on Goerli', async () => {
+      const paymentData = {
+        reference: '0xbeefaccc470c7dbd54de69',
+        txHash: '0xe472ca1b52751b058fbdaeaffebd98c0cc43b45aa31794b3eb06834ede19f7be',
+        from: '0x9c040e2d6fd83a8b35069aa7154b69674961e0f7',
+        to: '0x52e5bcfa46393894afcfe6cd98a6761fa692c594',
+        network: 'goerli',
+        salt: '0ee84db293a752c6',
+        amount: '92592592592592000',
+        requestId: '0188791633ff0ec72a7dbdefb886d2db6cccfa98287320839c2f173c7a4e3ce7e1',
+        block: 9945543,
+        token: '0x2bf02814ea0b2b155ed47b7cede18caa752940e6', //fDAIx
```

Comment:
I suggest that we move this address to a variable as we use this in some test cases.